docs/en/04_Changelogs/4.0.0.md

```diff
@@ -1491,8 +1491,8 @@ SilverStripe\ORM\DatabaseAdmin:
     GalleryPage: WebDesignGroup\ShopSite\GalleryPage 
 ```
 
-The next time you run a dev/build the class name for all `GalleryPage` pages will
-be automatically updated to the new `WebDesignGroup\ShopSite\GalleryPage`
+Once you've mapped the class names, you need to run the UpdateLegacyClassNamesTask (`/dev/tasks/UpdateLegacyClassNamesTask`) task.
+This will update affected columns and the `ClassName` field on each DataObject table will be substituted with the namespaced classname.
```

Comment:
```suggestion
This will update affected columns and the `ClassName` field on each DataObject table will be substituted with the new FQCN (fully qualified class name).
```